src/scripts/main.js

```diff
@@ -22,7 +22,17 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.innerText = key;
+    ul.appendChild(li);
```

Comment:
I believe, we still need that if, but a better one :)
Find the way how to detect if it's an object